### DISCUSSION TITLE
Access logs

### DISCUSSION THREAD

**Discussion by yeah-mike** [ggogel/seafile-containerized#20] [Q&A] [answered]
Hi,

I'm trying to add fail2ban to this deployment. This requires parsing the seafile log files.

Is there a way to access the log files?  I've added `accesslog = '/proc/1/fd/1'` to `gunicorn.conf.py` already. I thought perhaps the log files would be found in seafile-data, but I don't see any.

Thanks!

**Comment by ggogel** [comment, marked as answer]
Sorry for the late reply. When you set `accesslog = '/proc/1/fd/1'`, then the acceslog will be accessible through the `docker logs` command.
